buildSrc/build.gradle

```diff
@@ -70,6 +70,9 @@ allprojects {
   apply plugin: 'java'
   targetCompatibility = '11'
   sourceCompatibility = '11'
+  compileJava.options.encoding = 'UTF-8'
```

Comment:
Let's change this to configure _all_ compile tasks. Mainly because we also have a `compileMinimumRuntimeJava` task as well and we don't want to duplicate this three times and we should do the same to Groovy compilation tasks.

```
tasks.withType(AbstractCompile).configureEach {
    options.encoding = 'UTF-8'
}
```

Reply:
I changed it based on what @mark-vieira said.
- Add UTF-8 encoding in `compileMinimumRuntimeJava` task.
- Duplicate was removed.
- Checked that `error: unmappable character (0xA4) for encoding US-ASCII` does not appear.

Can you please review this again?